systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -104,7 +104,7 @@ public void testDeployUndeploy() {
     public void testKafkaConnectWithFileSinkPlugin() {
 
         String connectorConfig = getFileAsString("../systemtest/src/test/resources/file/sink/connector.json");
-        String kafkaConnectPodName = kubeClient.listResourcesByLabel("pod", "strimzi.io/type=kafka-connect").get(0);
+        String kafkaConnectPodName = kubeClient.listResourcesByLabel("pod", "type=kafka-connect").get(0);
```

Comment:
Why are the labels without the prefix? Is that a bug in the labels?

Reply:
After conversetion with @scholzj we found that I set custom `type: kafka-connect` in .yaml and this is a reason that test must use `"type=kafka-connect"`